app/models/concerns/image_file.rb

```diff
@@ -48,6 +67,17 @@ def url
     complete? ? self[:url] : self[:original_url]
   end
 
+  def href
+    complete? ? url : original_url
+  end
+
+  def href=(h)
```

Comment:
I started switching the 3 image APIs to match the Episode `media` API.  Where the main property an API consumer cares about is `href` - you PUT it to change an image url.  And once processing completes, it becomes the `f.prxu.org` CDN url of the copied image.